### PR TITLE
Crafter-BF 1.0 - React last Capstone

### PR DESCRIPTION
**Pull request by KaskMIL**
# In this new Pull Request I'm adding the following features:
- File structure, follow Dock pattern.
- Added Reducer's files to fetch the data from the APIs and the configStore file.
- Added a main page that contains the navigation through the different pages.
- Added a Styles page, that fetches the data from the API and displays the information, and added a filter that shows the data by ABV or IBU.
- Added a Hops page, that fetches the data from the API and displays the information, and added a filter that shows the data by Alpha Acid or  Beta Acid.
- Added a Yeast page, that fetches the data from the API and displays the information, and added a filter that shows the data by temperature: Fahrenheit or Celsius.
- Every page has a link to go back to the main page
- Added test for reducers and snapshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Crafter-BF
```
Comment (KDlamini) [inline README.md:4]: Kindly make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to Nelson Sakwa the author of the original design, as required by the Creative Commons License.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,67 @@
...
 
-### Deployment
+## 📝 License
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](./MIT.md) licensed.
```
Comment (KDlamini) [inline README.md:71]: Kindly add a working link to your license or otherwise remove it.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @KaskMIL  🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Redux actions, reducers, store,  setup correctly
✔️  Button events, navigation, and state management implemented
✔️  Application has integrated unit tests
✔️  Correct PR title
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KaskMIL** [comment]
Hi @KDlamini 
Thanks for your review, I will implement the changes suggeste
Have a great day, cheers!

**Review by medaminedev66 — APPROVED** [review]
Hi @KaskMIL 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- The repo looks professional
- Your app works as expected and runs without errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KaskMIL** [comment]
Hi @medaminedev66 
Thanks for your review!
Happy coding

**Pull request merged**